index.html

```diff
@@ -0,0 +1,124 @@
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+         <!---The second card -->
+         <div class="flex">
+            <img src="images/card2-image.png" alt="Image for Card 2">
+            <h2>TONIC</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+          <!---The Third card -->
+          <div class="flex">
+            <img src="images/card3-image.png" alt="Image for Card 3">
+            <h2>Multi-Post
+                Stories</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+           <!---The Fourth card -->
+           <div class="flex">
+            <img src="images/card4-image.png" alt="Image for Card 4">
+            <h2>TONIC</h2>
+           <div class="canopy">
+            <h3>CANOPY</h3>
+            <ul class="canopy-list"><li>BACK&nbsp;END&nbsp;Dev</li><li>2015</li></ul>
+           </div> 
+            <h4>
+                A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </h4>
+            <ul class="list second">
+            <li>html</li>
+            <li>css</li>
+            <li>javascript</li>
+            </ul>
+            <button class="see-more">See Project</button>
+        </div>
+
+    </section>
+    <!--About section-->
+    <section class="about">
+        <div class="about-content">
+            <h2>About <br> Myself</h2>
+            <p class="my-intro">Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what
+                 you see and have a project you need coded, 
+                don’t hestiate to contact me.
+            </p>
+            <br>
+            <p class="lets-connect-message">LET'S CONNECT</p>
```

Comment:
- [ ] Nice working on you about section, kindly make the `LET'S CONNECT` text a header tag `hx` for better use of HTML Semantics